static/js/settings.js

```diff
@@ -69,6 +69,12 @@ function setup_settings_label() {
         presence_enabled: $t({
             defaultMessage: "Display my availability to other users when online",
         }),
+        send_stream_typing_notifications: $t({
+            defaultMessage: "Display whether I'm typing messages in streams to other users",
+        }),
+        send_private_typing_notifications: $t({
+            defaultMessage: "Display whether I'm typing private messages to recipients",
+        }),
```

Comment:
It might be worthwhile to rephrase these but I'm unsure what would be nicer.

Reply:
It’s easy to misparse “to” as “messages to” rather than “display to”. I propose

* Let subscribers see when I’m typing messages in streams
* Let users see when I’m typing private messages to them
* Let users see when I’ve read private messages from them

Reply:
Those sound good to me. Changed.

> Let senders see when I’ve read private messages

Made this 'Let other users see when I've read messages' as we are allowing everyone who has access to the message to see read receipts.

Reply:
Did you mean to drop the “private”? And I think we need to be more specific than “other users”—maybe “participants”?

Reply:
Yes, as we show read receipts in streams too. (#18935 is the PR for that, and there's some discussion [at czo here](https://chat.zulip.org/#narrow/stream/101-design/topic/typing.20indicators.20and.20read.20receipts))

Changed 'other users' to 'participants'. It is more specific but sounds a bit odd to me (probably it's just me. I think we can stick to it).